### DISCUSSION TITLE
Kubernetes 1.21 Major Themes

### DISCUSSION THREAD

**Discussion by wilsonehusin** [kubernetes/sig-release#1436] [General]
Creating a space to submit major themes entries in release notes. We'd like to have folks submit the content over the release cycle instead of having them piled up at the end of the release cycle.

[Prior conversation on Slack](https://kubernetes.slack.com/archives/CN1KH4K9A/p1611602585021700) in which we agreed to use GitHub discussions.

Related discussion: [opt-in process for feature blog](https://kubernetes.slack.com/archives/C2C40FMNF/p1611168444088000)

**Comment by annajung** [comment]
It's too soon to tell at the moment, but adding a reference to the [enhancement tracking sheet](http://bit.ly/k8s121-enhancements) to let everyone know that it's available to take a quick peek at enhancements that are being worked on for the release

**Comment by wilsonehusin** [comment]
SIG Auth: [PodSecurityPolicy Deprecation][psp-pr] we will probably need to link it up with [the corresponding blog on this matter][psp-blog]

[psp-pr]: https://github.com/kubernetes/kubernetes/pull/97171
[psp-blog]: https://github.com/kubernetes/website/issues/26202

**Comment by sftim** [comment]
Kubernetes' API reference documentation is moving to https://kubernetes.io/docs/reference/kubernetes-api/

See https://github.com/kubernetes/website/issues/25505

**Comment by eddiezane** [comment]
The version of [kustomize](https://github.com/kubernetes-sigs/kustomize) that ships with kubectl has been upgraded from v2.0.3 to v4.0.5. A major part of this change is that kustomize is now treated as a library so future upgrades will happen more often.

https://github.com/kubernetes/kubernetes/pull/98946

cc @monopole @seans3 @natasha41575

**Reply by KnVerey** [reply]
Also for SIG-CLI: A new `kubectl.kubernetes.io/default-container` annotation can be set on pods to preselect a container for kubectl commands. https://github.com/kubernetes/enhancements/issues/2227 @pacoxu

**Reply by KnVerey** [reply]
One more for SIG-CLI: Alpha implementation of Kubectl Command Headers: set `KUBECTL_COMMAND_HEADERS=1` to have `kubectl` send additional http request headers with context about the command that created the request. https://github.com/kubernetes/enhancements/issues/859 @seans3

**Comment by wilsonehusin** [comment]
Per @neolit123 [in Slack](https://kubernetes.slack.com/archives/C13J86Z63/p1616433444006000?thread_ts=1616078555.000700&cid=C13J86Z63), SIG Cluster Lifecycle / Kubeadm does not have any entries.

**Comment by wilsonehusin** [comment]
SIG Autoscaling also has none, [per conversation on Slack](https://kubernetes.slack.com/archives/C09R1LV8S/p1616078532016100).

**Comment by xing-yang** [comment]
Major themes for SIG-Storage are in this doc: https://docs.google.com/document/d/1HZXiokkq8r7R9R8O-yQy1cRaEGzvqdkqmwojBrguyZU/edit?usp=sharing
CC @saad-ali @msau42 @jsafrane
